guide/src/StyleGuide.js

```diff
@@ -12,7 +12,6 @@ import ComponentExList from './ComponentExList';
 import ThemeExList from './ThemeExList';
 import { Header, SideNav, Figure, ThemeExamples } from './components';
 import MarkdownElement from './components/MarkdownElement';
-import Code from './components/Code';
```

Comment:
This should have not been added in https://github.com/cyverse/cyverse-ui/pull/41. Is a result of doing too many things on a branch and then trying to sort it all out with rebase.